node/overseer/overseer-gen/proc-macro/src/impl_builder.rs

```diff
@@ -35,6 +68,12 @@ pub(crate) fn impl_builder(info: &OverseerInfo) -> proc_macro2::TokenStream {
 		.iter()
 		.map(|subsystem_name| format_ident!("{}_with", subsystem_name))
```

Comment:
where is it used?

Reply:
I think this can be removed again in a future PR, this was added when I was under the impression that we could get away without `OverseerConnector` which was wrong ultimately.

I'll remove this in a separate PR.